SimCalorimetry/HcalSimAlgos/src/HcalSiPMShape.cc

```diff
@@ -20,7 +20,7 @@ void HcalSiPMShape::computeShape(unsigned int signalShape) {
   //grab correct function pointer based on shape
   double (*analyticPulseShape)(double);
   if(signalShape==HcalShapes::ZECOTEK || signalShape==HcalShapes::HAMAMATSU) analyticPulseShape = &HcalPulseShapes::analyticPulseShapeSiPMHO;
-  else if(signalShape==HcalShapes::HE2017) analyticPulseShape = &HcalPulseShapes::analyticPulseShapeSiPMHE;
```

Comment:
keep the old shape here as well (can be `else if(signalShape==HcalShapes::HE2017 or signalShape==HcalShapes::HE2018)` to simplify)